projects/jmeter.apache.org/package.yml

```diff
@@ -11,6 +11,9 @@ warnings:
   - vendored
 dependencies:
   openjdk.org: '*'
+runtime:
+  env:
+    JAVA_HOME: "{{prefix}}"
```

Comment:
This seems wrong, the user can set this, we are overriding it.

Reply:
Hmm 🤔 

perhaps it would be correct to check the existence of a variable and assign a value if it does not exist?

Reply:
why is it set? Add comments pls

Reply:
JAVA_HOME just resolve problem 
```
The operation couldn’t be completed. Unable to locate a Java Runtime.
Please visit http://www.java.com/ for information on installing Java.
````
https://github.com/teaxyz/pantry/actions/runs/5743363132/job/15567693928

Reply:
Or maybe we can add JAVA_HOME in `test` section and add `README.md` with information that JAVA_HOME must be set in env?

Reply:
Shouldn't `JAVA_HOME` be `${{deps.openjdk.org.prefix}}`, and exported from that package? Per my limited recollection of configuring Java. @ABevier was a Javanista: any thoughts on this?

Reply:
Yes, the final version is {{deps.openjdk.org.prefix}}

https://github.com/teaxyz/pantry/commit/70957af471f223d8cb60abf22867b1e6cd0e2a04

Reply:
Right. But shouldn't that be in the runtime of `openjdk.org`, not packages that depend on it?

Reply:
> Right. But shouldn't that be in the runtime of `openjdk.org`, not packages that depend on it?

Yes, I had thoughts of doing this too. Then I will open a pull request for openjdk and add JAVA_HOME